.travis.yml

```diff
@@ -0,0 +1,13 @@
+---
+sudo: false
+language: ruby
+rvm:
+  - 2.0.0
```

Comment:
Not planning on it. Azure SDK for ruby https://github.com/Azure/azure-sdk-for-ruby is supported for Ruby 2+, so I'm following the same here.